interfaces/python/subsample/__init__.py.in

```diff
@@ -5,6 +5,7 @@ import numpy as np
 
 @dataclass
 class PoissonSampling:
+    """Provides access to Poisson subsampling by creating an instance of that class and calling subsample() on it."""
```

Comment:
I'm not sure what's the best doc style for data-classes in Python since there seem to be many ways, but documentation for the single properties should also be added here so that you see it in Python.

Reply:
Personally, I like [google style doc strings](https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html#example-google). For data classes, I would use the `Attributes` rather than the `Args` section, but no strong oppionion here ...